src/styles/index.css

```diff
@@ -0,0 +1,95 @@
+@import "./reset.css";
+@import "./stars.css";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
```

Comment:
width is an external style 
<img width="999" alt="image" src="https://user-images.githubusercontent.com/102411130/201629997-e32fd0de-6a12-45b8-b330-eab2d703b3fd.png">

and check css properties order
http://codeguide.academy/html-css.html#css-order